ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -207,61 +176,49 @@ public void onInitializeSuccess() {
           public void onInitializeError(String errorMessage) {
             Log.d(TAG, "SDK init failed: " + VungleBannerAdapter.this);
             VungleManager.getInstance().removeActiveBannerAd(mPlacementId);
-            VungleListener listener = mVungleListener.get();
-            if (mPendingRequestBanner && listener != null) {
-              listener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
+            MediationBannerListener listener = bannerListener.get();
+            if (mPendingRequestBanner && listener != null && adapter.get() != null) {
```

Comment:
minor can it be garbage collected between 2 bannerListener.get() calls?

Reply:
perhaps...if move back, should adding vungleListener and make VungleIntestitialAdapter clean break.

Reply:
no, just get it before and store before if and call like listener

Reply:
done.